.gitignore

```diff
@@ -30,3 +30,4 @@ coverage.xml
 docs/_build/
 docs/_templates/
 cover
+sandbox
```

Comment:
and also is it something you can put in your global git config? not sure if it's project-related.